apps/pr-deploy-site/index.html

```diff
@@ -118,7 +118,12 @@ <h1>Deployed sites for PR <a href="#" class="prLink"></a></h1>
       </li>
       <li class="Tile">
         <a href="./fabric-website/dist/index.html?devhost" class="Tile-link"
-          ><i class="ms-Icon ms-Icon--Website"></i>Web Site</a
+          ><i class="ms-Icon ms-Icon--Website"></i>Website</a
+        >
+      </li>
+      <li class="Tile">
+        <a href="./react-northstar/index.html" class="Tile-link"
```

Comment:
```suggestion
        <a href="./react-northstar" class="Tile-link"
```

As we use exact matching:
```
<Route exact path="/" component={Introduction} />
```